apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -476,19 +498,19 @@ log_netns_diff(Namespaces, _PrevNamespaces) ->
 -spec(handle_netns_event(EventType, [netns()], state()) -> state()
     when EventType :: add_netns | remove_netns | reconcile_netns).
 handle_netns_event(remove_netns, ToDel,
-        #state{ipvs_mgr=IPVSMgr, route_mgr=RouteMgr, netns=Prev}=State) ->
+        #state{ipvs_mgr=IPVSMgr, route_mgr=RouteMgr, namespaces=Prev}=State) ->
     Namespaces = dcos_l4lb_route_mgr:remove_netns(RouteMgr, ToDel),
-    Namespaces = dcos_l4lb_route_mgr:remove_netns(IPVSMgr, ToDel),
+    Namespaces = dcos_l4lb_ipvs_mgr:remove_netns(IPVSMgr, ToDel),
```

Comment:
😱 This appeared to be a serious bug!!

Reply:
Most likely the process crashes in this case and supervisor restarts it. I think it's not a big deal.

Reply:
wouldn't we miss removing IPVS configuration?

Reply:
If I understand the code correctly, we remove network namespace from manager processes when it already not longer exists.